docs/extra.js

```diff
@@ -0,0 +1,5 @@
+window.intercomSettings = {
+        app_id: "itolwzo2"
+    };
+
+    (function () { var w = window; var ic = w.Intercom; if (typeof ic === "function") { ic('reattach_activator'); ic('update', w.intercomSettings); } else { var d = document; var i = function () { i.c(arguments); }; i.q = []; i.c = function (args) { i.q.push(args); }; w.Intercom = i; var l = function () { var s = d.createElement('script'); s.type = 'text/javascript'; s.async = true; s.src = 'https://widget.intercom.io/widget/' + "itolwzo2"; var x = d.getElementsByTagName('script')[0]; x.parentNode.insertBefore(s, x); }; if (w.attachEvent) { w.attachEvent('onload', l); } else { w.addEventListener('load', l, false); } } })();
```

Comment:
nit: GitHub is reporting a missing new line at the end of the file. Mis-configured IDE, maybe?

Reply:
@daceynolan in case you use VSCode: https://stackoverflow.com/a/44704969/4716370

Reply:
To be clear, I'm talking about this here:
![image](https://user-images.githubusercontent.com/3299086/100278840-c4822d00-2f65-11eb-8dc3-9b3e909ef87a.png)